docs/usage/shooted_seed.md

```diff
@@ -0,0 +1,73 @@
+# Shooted Seeds
+
+Create shooted seeds with the `shoot.gardener.cloud/use-as-seed` annotation
+
+## Preparation
+
+### Edit `garden` namespace
+
+The namespace "garden" needs 2 new labels for the project:
+
+```yaml
+labels:
+  gardener.cloud/role: project
+  project.gardener.cloud/name: garden
+```
+
+### `garden` Project
+
+The annotation works only for `Shoot`s created in the `garden` namespace. Consequently, we have to create a project for the `garden` namespace first using `kubectl` (we don't use the Gardener Dashboard as it would add a `garden` prefix for the namespace).
+
+Example: [/example/05-project-dev.yaml](../../example/05-project-dev.yaml)
+
+```yaml
+apiVersion: core.gardener.cloud/v1beta1
+kind: Project
+metadata:
+  name: garden
+spec:
+  owner:
+...
+  namespace: garden
+```
+
+## Create shoot
+
+### Deploy shoot
+
+Change the networks for the shooted seed that it is not overlapping with his shoots (spec.provider.infrastructureConfig.networks and spec.networking).
+Fullexample: [/example/90-shoot.yaml](../../example/90-shoot.yaml)
+
+Set the annotation for the shooted seed.
+Example:
+
+```yaml
+  annotations:
+    shoot.gardener.cloud/use-as-seed: >-
+      true,shootDefaults.pods=100.96.0.0/11,shootDefaults.services=100.64.0.0/13,disable-capacity-reservation,with-secret-ref
+```
+
+This annotation contains a few configurations for the seed:
+Option | Description
+--- | ---
+`true` | deploys the gardenlet into the shoot which will automatically register the cluster as seed
+`no-gardenlet` | prevents the automatic deployment of the gardenlet into the shoot. Instead, the `Seed` object will be created with the assumption that another gardenlet will be responsible for managing it (according to its `seedSelector` configuration).
+`disable-capacity-reservation` | set `spec.settings.excessCapacity.enabled` in Seed to false (see [/example/50-seed.yaml](../../example/50-seed.yaml))
+`invisible` | set `spec.settings.scheduling.visible` in Seed to false  (see [/example/50-seed.yaml](../../example/50-seed.yaml))
```

Comment:
Can we either always write `Seed` and `Shoot` in capital letters or always in small letters? There seem to be some irregularities.